pkg/yamlSchemaValidator/validator_test.go

```diff
@@ -0,0 +1,112 @@
+package yamlSchemaValidator
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/datreeio/datree/pkg/fileReader"
+	"github.com/datreeio/datree/pkg/policy"
+	"gopkg.in/yaml.v2"
+)
+
+type TestFilesByRuleId = map[int]*FailAndPassTests
+
+type FailAndPassTests struct {
+	fails  []*FileWithName
+	passes []*FileWithName
+}
+
+type FileWithName struct {
+	name    string
+	content string
+}
+
+func TestDefaultRulesValidation(t *testing.T) {
```

Comment:
The common convention for test function is TestXxx where Xxx is the name of the tested function/method. I suggest renaming this test because it might be confusing for the next developer because there isn't any `DefaultRulesValidation` function. IMHO, the name should indicate `Validate` and testing the default rules is one of the test cases.

Reply:
renamed to `TestValidate`